doc/nest-03.md

```diff
@@ -0,0 +1,35 @@
+# Installing and uninstalling a package
+
+## Nest pull
+
+`nest pull` synchronizes the local packages' cache with the one on Raven's server. You need to **keep this local cache up to date as often as possible**, or you won't be able to install the packages or versions that came out after your last cache update. This is very important. Do not forget that.
```

Comment:
`install the packages or versions that came out` -> `install the packages or updates that came out`